### PR TITLE
GUI/sound design/182

### PR DESCRIPTION
**Pull request by GuyomT**
Adding sounds for events spawn death egg win or loose, not all played cause GUI have not yet access for win or lose information

### CODE REVIEW

**Review by LucasTesnier — APPROVED** [review]
NTR!

File: src/zappy_gui_src/Core/SFML/Map/Map.cpp [hunk 1]
```diff
@@ -297,4 +330,6 @@ void Map::display()
     }
     _displaySelectedTile();
     _displayHoveredTile();
+    // if (_sounds.at(0)->getStatus() != sf::Sound::Playing)
```
[the remark this answers is not in the document]
Comment (GuyomT) [reply src/zappy_gui_src/Core/SFML/Map/Map.cpp:333]: indeed, a test i forgot to remove

**Review by osvegn — APPROVED** [review]
OK

**Review by pariciadra — APPROVED** [review]
Lgtm

**Pull request merged**